xarray/backends/h5netcdf_.py

```diff
@@ -390,7 +387,6 @@ def open_dataset(
             use_cftime=use_cftime,
             decode_timedelta=decode_timedelta,
         )
-        return ds
```

Comment:
I don't _really_ care, but I do agree with @alexamici that the separate `return` at the end is a little clearer.